pathway-analysis/qusage_meta_analysis.Rmd

```diff
@@ -220,8 +238,10 @@ rm(northcott_exprs_df)
 #### Metadata
 
 ```{r}
-northcott_metadata_file <- file.path("data", "GSE37382", 
-                                     "metadata_GSE37382.tsv")
+northcott_metadata_file <- file.path(data_dir,
+                                     "GSE37382", # Replace with name of your unzipped folder
```

Comment:
This wording should also be changed if we don't think users would zip their data, which I don't think they would.